lib/mayaUsd/ufe/wrapUtils.cpp

```diff
@@ -76,28 +77,7 @@ std::string stagePath(UsdStageWeakPtr stage)
 
 UsdPrim ufePathToPrim(const std::string& ufePathString)
 {
-    // The path string is a list of segment strings separated by ',' comma
-    // separator.
-    auto segmentStrings = TfStringTokenize(ufePathString, ",");
-
-    // If there's just one segment, it's the Maya Dag path segment, so it can't
-    // have a prim.
-    if (segmentStrings.size() == 1) {
-        return UsdPrim();
-    }
-
-    // We have the path string split into segments.  Build up the Ufe::Path one
-    // segment at a time.  The path segment separator is the first character
-    // of each segment.  We know that USD's separator is '/' and Maya's
-    // separator is '|', so use a map to get the corresponding UFE run-time ID.
-    Ufe::Path                        path;
-    static std::map<char, Ufe::Rtid> sepToRtid = { { '/', ufe::getUsdRunTimeId() }, { '|', 1 } };
-    for (std::size_t i = 0; i < segmentStrings.size(); ++i) {
-        const auto& segmentString = segmentStrings[i];
-        char        sep = segmentString[0];
-        path = path + Ufe::PathSegment(segmentString, sepToRtid.at(sep), sep);
-    }
-    return ufe::ufePathToPrim(path);
+    return ufe::ufePathToPrim(Ufe::PathString::path(ufePathString));
```

Comment:
Curious - is this related to the change or simply cleanup?

Reply:
Cleanup.  Saw this, couldn't resist.